src/pretix/control/views/modelimport.py

```diff
@@ -64,28 +68,16 @@
 )
 
 
-class ImportView(EventPermissionRequiredMixin, TemplateView):
-    template_name = 'pretixcontrol/orders/import_start.html'
-    permission = 'can_change_orders'
-
+class BaseImportView(TemplateView):
     def post(self, request, *args, **kwargs):
         if 'file' not in request.FILES:
-            return redirect_to_url(reverse('control:event.orders.import', kwargs={
-                'event': request.event.slug,
-                'organizer': request.organizer.slug,
-            }))
+            return redirect_to_url(request.path)
         if not request.FILES['file'].name.lower().endswith('.csv'):
```

Comment:
Should we allow .txt files as well?

Reply:
I think that only leads to additional confusion?

Reply:
You are probably right!